ebos/ecloutputblackoilmodule.hh

```diff
@@ -1,4 +1,4 @@
-// -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
```

Comment:
this change seems unintentional.

Reply:
Yes - this was unintentional (sorry)- has now been removed!